specs/capella/light-client/full-node.md

```diff
@@ -76,3 +76,31 @@ def block_to_light_client_header(block: SignedBeaconBlock) -> LightClientHeader:
         execution_branch=execution_branch,
     )
 ```
+
+## Deriving light client data
+
+### `create_light_client_update`
+
+Full nodes SHOULD continue deriving the `LightClientUpdate` for each sync committee period as per the altair specs but with the modified or upgraded `attested_header` and `finalized_header` with the following considerations:
```

Comment:
Hmm. The way how `consensus-specs` work is that the later forks are overlays. So, the phase0 spec is used as basis, then altair is overlayed on it (adding new functions, and replacing redefined ones), then bellatrix is overlayed, then capella. 

So, technically, the "altair specs" being referred here are also copied to capella as is, with `block_to_light_client_header` getting overridden.

That's sort of what the "introduction" above mentions:
> It extends the [Altair Light Client specifications](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/sync-protocol.md).